Scripts/Proxies/Data/SerializedProxyFeedback.cs

```diff
@@ -0,0 +1,124 @@
+﻿#if UNITY_EDITOR
+using System;
+using UnityEngine;
+using UnityEngine.InputNew;
+
+namespace UnityEditor.Experimental.EditorVR.Proxies
+{
+    [Serializable]
+    class SerializedProxyFeedback
+    {
+        public SerializedProxyNodeFeedback leftNode;
+        public SerializedProxyNodeFeedback rightNode;
+    }
+
+    /// <summary>
+    /// Used as globally unique identifiers for feedback requests
+    /// They are used to relate feedback requests to the persistent count of visible presentations used to suppress feedback
+    /// </summary>
+    [Serializable]
+    class RequestKey
+    {
+        /// <summary>
+        /// The control index used to identify the related affordance
+        /// </summary>
+        [SerializeField]
+        VRInputDevice.VRControl m_Control;
+
+        /// <summary>
+        /// The tooltip text that was presented
+        /// </summary>
+        [SerializeField]
+        string m_TooltipText;
+
+        public void UpdateValues(ProxyFeedbackRequest request)
+        {
+            m_Control = request.control;
+            m_TooltipText = request.tooltipText;
+        }
+
+        public bool HasTooltip()
+        {
+            return !string.IsNullOrEmpty(m_TooltipText);
+        }
+
+        public override int GetHashCode()
+        {
+            var hashCode = (int)m_Control;
+
+            if (m_TooltipText != null)
+                hashCode ^= m_TooltipText.GetHashCode();
+
+            return hashCode;
+        }
+
+        public override bool Equals(object obj)
+        {
+            if (obj == null)
+                return false;
+
+            if (!(obj is RequestKey))
+                return false;
+
+            var key = (RequestKey)obj;
+            return m_Control == key.m_Control && string.Equals(m_TooltipText, key.m_TooltipText);
+        }
+
+        public override string ToString()
+        {
+            return string.Format("{0}: {1}", m_Control, m_TooltipText);
+        }
+    }
+
+    /// <summary>
+    /// Contains per-request persistent data
+    /// </summary>
+    [Serializable]
+    class RequestData
+    {
+        readonly Action m_OnBecameVisible;
```

Comment:
Move below properties

Reply:
This is addressed. Not sure why it's not hidden.